pkg/components/calico.go

```diff
@@ -131,11 +131,28 @@ var (
 	}
 
 	ComponentCalicoGoldmane = Component{
-		Version:  "master",
+		Version:  "bmv1.10",
 		Image:    "calico/goldmane",
-		Registry: "",
+		Registry: "gcr.io/unique-caldron-775/brianmcmahon/",
```

Comment:
Need to regenerate - using your versions still

Reply:
done